code/game/objects/items/storage/boxes/debug_boxes.dm

```diff
@@ -0,0 +1,193 @@
+/**
+ * Basically the boxes that spawn in debug modsuit
+ */
+
+/obj/item/storage/box/material
+	name = "box of materials"
+	illustration = "implant"
+
+/obj/item/storage/box/material/PopulateContents() //less uranium because radioactive
+	// amount should be null if it should spawn with the type's default amount
```

Comment:
have you checked this still spawns correctly, that null value will still get passed as an arg